src/Components/NavBar/Menus/MobileNavMenu.tsx

```diff
@@ -61,10 +55,14 @@ export const MobileNavMenu: React.FC = () => {
         </>
       ) : (
         <>
-          <MobileLink onClick={() => authentication.login(mediator)}>
+          <MobileLink
+            href={`/log_in?intent=signup&signupIntent=signup&trigger=click&contextModule=Header`}
+          >
             Login
           </MobileLink>
-          <MobileLink onClick={() => authentication.signup(mediator)}>
+          <MobileLink
+            href={`/sign_up?intent=signup&signupIntent=signup&trigger=click&contextModule=Header`}
+          >
```

Comment:
These are actually direct links. Grabbed urls old nav bar [here](https://github.com/damassi/force/blob/8b7bdfa6f9b0a7cc8c9b7bd7bd2b38e2c3a1ed7c/src/mobile/components/layout/templates/login-signup.jade#L1).

Reply:
Is there any learning from here that we should apply other places? Like...I know we use the mediator to do logins many places, why are these different? And is there a documented heuristic for when we should use the mediator to handle logins vs navigating to a specific URL?

Reply:
So how we currently have it is there's the `/desktop` app and then there's `/mobile` (m.artsy), each with subtly different flows. When we rebuilt the logins a little over a year ago we built in full-page route-based login screen vs a popup, due to the small screens. Before this change I had made the mistake of assuming everything was now totally responsive but when a mobile useragent is detected, for many screens it will do a hard redirect to the mobile app, where these routes then kick in.

Reply:
Thank you for the explanation! 🙏